packages/next/src/server-build.ts

```diff
@@ -1112,6 +1113,24 @@ export async function serverBuild({
               }
               break;
             }
+            case 'server/app-paths-manifest.json': {
+              console.log(normalizedPages);
```

Comment:
maybe worth keeping this behind the debug flag along with the other log

Reply:
oh these can be removed now